docs/overview/draw.rst

```diff
@@ -295,7 +295,7 @@ documentation.
         const lv_point_t * p2;        /**< A point calculated during drawing. E.g. a point of chart.*/
         char * text;                  /**< A text calculated during drawing. Can be modified. E.g. tick labels on a chart axis.*/
         uint32_t text_length;         /**< Size of the text buffer containing null-terminated text string calculated during drawing.*/
-        uint32_t part;                /**< The current part for which the event is sent*/
+        lv_part_t part;                /**< The current part for which the event is sent*/
```

Comment:
This page of the docs is for v8 and outdated and [this](https://docs.lvgl.io/master/porting/draw.html#draw-task) is used instead.  Please remove this file instead, and we can add a new one later.